Cargo.toml

```diff
@@ -19,6 +19,9 @@ rust-version = "1.36"
 # one case, and only f64) at the cost of some performance.
 small = []
 
+# Enable to `Number.prototype.toFixed()` float to string conversion.
+to-fixed = []
+
```

Comment:
Should this be a feature? The compiler is pretty smart at detecting dead code, so an user who doesn't want to use `to_fixed` and doesn't want to incur the code size cost can just... not use the function.

Reply:
Yes, but it also causes the Buffer to increase in size, that's why it was made a feature, we could have a separate buffer tho...

Reply:
But the increase in buffer size should be barely noticeable, right? only 100 bytes more. And if in the future we conclude that the size impact is noticeable, we can always specialize using some phantom types and generics.